client/src/Components/Home.js

```diff
@@ -0,0 +1,172 @@
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import styles from "../Styles/Home.module.css";
+import { motion, AnimatePresence } from "framer-motion";
+import pork from "../Images/Pork-Butt2.jpg";
+import brisket from "../Images/Brisket2.jpg";
+import frozen from "../Images/brisketsealed.jpg";
+import fresh from "../Images/freshsmoked.jpg";
+
+const Home = () => {
+  const [intro, setIntro] = useState(true);
+  const [stepOne, setStepOne] = useState(false);
+  const [stepTwo, setStepTwo] = useState(false);
+  const navigate = useNavigate();
+
+  const handleClick = () => {
+    setIntro(!intro);
+    setStepOne(true);
+  };
+  const handleClickTwo = () => {
+    setStepOne(!stepOne);
+    setStepTwo(true);
+  };
```

Comment:
this can be handled better if you use a `steps` state.

ex 
```
const handleClick = (step) => {
   setStep(step)
}

```

where `step` can be "intro", "one", "two" , etc...

Reply:
done